libs/wire-api/test/unit/Test/Wire/API/MLS.hs

```diff
@@ -62,7 +62,7 @@ tests =
       testCase "key package ref" testKeyPackageRef,
       testCase "validate message signature" testVerifyMLSPlainTextWithKey,
       testCase "create signed remove proposal" testRemoveProposalMessageSignature,
-      testCase "parse GroupInfoBundle" testParseGroupInfoBundle
+      testCase "parse GroupInfoBundle" testParseGroupInfoBundle -- TODO: remove this also
```

Comment:
Leftover TODO?

Reply:
Since @smatting is not here to clarify, I'm unsure whether to make this `futurework` or disable this test. It is currently green for me anyway…